src/schema/objects/metadata.yaml

```diff
@@ -561,6 +561,36 @@ DecayCorrectionFactor:
   type: array
   items:
     type: number
+DeidentificationMethod:
+  name: DeidentificationMethod
+  display_name: Deidentification Method
+  description: |
+    A description of the mechanism or method used to remove the Patient's identity.
+    Corresponds to DICOM Tag 0012, 0063 `De-identification Method`.
+  type: string
+DeidentificationMethodCodeSequence:
+  name: DeidentificationMethodCodeSequence
+  display_name: Deidentification Method Code Sequence
+  description: |
+    A description of the mechanism or method used to remove the Patient's identity.
+    Corresponds to DICOM Tag 0012, 0064 `De-identification Method Code Sequence`.
+  type: array
+  items:
+    type: object
+    recommended_fields:
+      - CodeValue
+      - CodeMeaning
+      - CodingSchemeDesignator
+      - CodingSchemeVersion
+    properties:
+      CodeValue:
+        type: string
+      CodeMeaning:
+        type: string
+      CodingSchemeDesignator:
+        type: string
+      CodingSchemeVersion:
+        type: string
```

Comment:
Fleshing out these values with further descriptions, so we can make a table describing these fields.

```suggestion
      - CodeValue
      - CodeMeaning
      - CodingSchemeDesignator
      - CodingSchemeVersion
    properties:
      CodeValue:
        name: CodeValue
        type: string
        description: |
          An identifier that is unambiguous within the Coding Scheme
          denoted by Coding Scheme Designator and Coding Scheme Version.
          Corresponds to DICOM Tag 0008, 0100 `Code Value`.
      CodeMeaning:
        name: CodeMeaning
        type: string
        description: |
          Text that has meaning to a human and conveys the meaning of the term
          Corresponds to DICOM Tag 0008, 0104 `Code Meaning`.
      CodingSchemeDesignator:
        name: CodingSchemeDesignator
        type: string
        description: |
          The identifier of the coding scheme in which the coded entry is defined.
          Corresponds to DICOM Tag 0008, 0102 `Coding Scheme Designator`.
      CodingSchemeVersion:
        name: CodingSchemeVersion
        type: string
        description: |
          An identifier of the version of the coding scheme if necessary to resolve ambiguity.
          Corresponds to DICOM Tag 0008, 0103 `Coding Scheme Version`.
```